subsys/bluetooth/host/conn_internal.h

```diff
@@ -126,8 +126,13 @@ struct bt_conn_iso {
 	};
 
 #if defined(CONFIG_BT_ISO_UNICAST) || defined(CONFIG_BT_ISO_BROADCASTER)
-	/** 16-bit sequence number that shall be incremented per SDU interval */
-	uint16_t seq_num;
+	/** @brief 16-bit sequence number that shall be incremented per SDU interval
```

Comment:
Is it a good thing to have this "shall" here?
This is the struct, is this the right place to mandate behavior?  And if we do, should it say who is responsible to increment it?
Propose to end sentence after sequence number".

Reply:
I think it's fine to keep it in this (internal) struct, as that will serve as a comment for developers if they attempt to change how the `seq_num` is handled